setup.py

```diff
@@ -49,13 +49,14 @@ def read(*rnames):
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope3'],
-    url='http://pypi.python.org/pypi/z3c.authviewlet',
+        'Framework :: Zope :: 3'],
```

Comment:
We have "Framework :: Zope3" in 209 zopefoundation packages.

We have "Framework :: Zope :: 3" in 3 packages (z3c.checkversions, zodbupdate and zope.fanstatic).

I think we should stick with the more popular classifier.

Reply:
The more popular classifier is the older one which I am going to declare as deprecated. (To get a Trove classifier for Zope 4 we had to switch to the new schema.) My plan is to migrate all usages of the old classifier to the new format. My idea was to do this every time I see an old classifier.

I do not have checkouts for all 209 repositories so I would appreciate it very much if you could
help with a mass commit to make the migration faster. I do not think that we will need releases right now. For me it is enough that the packages will be released eventually when it is needed for a feature or bugfix.

Reply:
I would like some meta-repository where we could track cross-repo issues like this.  https://github.com/zopefoundation/zopefoundation/issues maybe?

Reply:
Good idea. I decided for `meta` as project name, so there is now zopefoundation/meta#1.